brave/brave-ios
# Fix #6003:  Include option for users to hide privacy stats from the NTP

Adding Functionality for hide NTP favourites and Privacy Hub
Adding UIAction menu for opening and hiding Hub
Add new settings design for New TP



## Summary of Changes


This pull request fixes #6003

## Submitter Checklist:


## Test Plan:

- Press 3 dot icon on Privacy Hub and test open - hide
- Go settings -> new tab page 
- Check new design with inset grouped list
- Test privacy hub and favourites toggles and check if toggle make the proper section visible and hid it

## Screenshots:


![test1 1](https://user-images.githubusercontent.com/6643505/191630620-62dec3f1-2b5e-4139-a68e-32ee529aa551.png)


https://user-images.githubusercontent.com/6643505/191630638-e96decc2-e553-4d0e-aed8-7ddefd978bc0.MP4


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
